core/src/test/java/jenkins/Junit4TestsRanTest.java

```diff
@@ -0,0 +1,13 @@
+package jenkins;
+
+import org.junit.Test;
+
+class Junit4TestsRanTest {
+
+    @Test
+    void anything() {
+        // intentionally blank.
+        // we just want a test that runs with junit 4 so that if tests get skipped due to the introduction of ju[iter engine we are allerted
```

Comment:
```suggestion
        // we just want a test that runs with junit 4 so that if tests get skipped due to the introduction of jupiter engine we are alerted
```